com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/ezsp/EzspFrameRequest.java

```diff
@@ -40,16 +41,13 @@
  *
  */
 public abstract class EzspFrameRequest extends EzspFrame {
-    private final static AtomicLong sequence = new AtomicLong(1);
+    private final static AtomicInteger sequence = new AtomicInteger(1);
 
     /**
      * Constructor used to create an outgoing frame
      */
     protected EzspFrameRequest() {
-        sequenceNumber = (int) sequence.getAndIncrement();
-        if (sequenceNumber == 254) {
-            sequence.set(1);
-        }
+        sequenceNumber = sequence.getAndUpdate(n -> n <254 ? n+1 : 1);
```

Comment:
Please use the old notation and synchronise the method for backward compatibility.